models/webhook/hooktask.go

```diff
@@ -100,7 +100,7 @@ type HookResponse struct {
 // HookTask represents a hook task.
 type HookTask struct {
 	ID              int64 `xorm:"pk autoincr"`
-	RepoID          int64 `xorm:"INDEX"`
+	RepoID          int64 `xorm:"INDEX"` // unused
```

Comment:
I think this filed should be removed to prevent from misuse.

Reply:
I removed it first but did you read https://github.com/go-gitea/gitea/pull/17940#discussion_r766272629 ?

Reply:
Hmm ... I had read it before, unfortunately, I forget again. 😭

So I think we can add more comments about why it is still here. I believe in future there will be other developers ask the same question ...........

Reply:
I would remove it and add it back if we need/want it in the future. If we want to add a reference to the source the name `RepoID` will be wrong and the fields needs a new name anyway.